priv/samples/tflite.livemd

```diff
@@ -0,0 +1,143 @@
+# Use TensorFlow Lite on Nerves Livebook
+
+```elixir
+IO.puts("Mix.install/2 is not supported on Nerves Livebook (yet!).")
+```
+
+## Introduction
+
+TensorFlow Lite is a stripped-down version of
+[TensorFlow](https://en.wikipedia.org/wiki/TensorFlow), a free and open-source
+software library for machine learning and artificial intelligence.
+
+In Elixir, we can use TensorFlow Lite through the package
+[`tflite_elixir`](https://github.com/cocoa-xu/tflite_elixir), which does the
+TensorFlow Lite Elixir bindings with optional [Edge
+TPU](https://en.wikipedia.org/wiki/Tensor_Processing_Unit#Edge_TPU) support.
+
+In this notebook, we will perform image classification with pre-trained model.
+
+When working with TensorFlow Lite, we make extensive use of the functions from
+TFLiteElixir, so it's useful to alias the module as something shorter.
+
+```elixir
+alias TFLiteElixir, as: TFLite
+alias TFLiteElixir.Interpreter
+```
+
+## Download model and labels
+
+It is worth mentioning that the only writable partition on a Nerves device, by
+default, is the `/root` partition, which is also symlinked as `/data`.
+
+```elixir
+use Toolshed
+cmd("ls -al /root")
+```
```

Comment:
Nerves has other writable partitions. For example, `/tmp` is a writable partition, but it's in-memory and might not be big enough on some devices. I would expect `/tmp` to work on a Pi4, though. I think that it's best to delete this and I'll add some text below for downloading to a temporary directory.

```suggestion
```